monai/handlers/mlflow_handler.py

```diff
@@ -99,6 +99,8 @@ class MLFlowHandler:
 
     # parameters that are logged at the start of training
     default_tracking_params = ["max_epochs", "epoch_length"]
+    # the finish status of a mlflow run
+    run_finish_status = mlflow.entities.RunStatus.to_string(mlflow.entities.RunStatus.FINISHED)
```

Comment:
`mlflow` is a optional import. This will force to load the mlflow when python is checking the class defs before instantiation, which could be the cause of failures in the min tests (which does not install dev packages).

Reply:
Thanks @mingxin-zheng! This helps a lot and solved the issue.